source/models/transcript.rst

```diff
@@ -43,9 +43,10 @@ confidence             double           confidence score of the provided value.
 
 When a job is submitted with option :ref:`skip_diarization <options-model>` set to ``false`` or is not specified, the transcript will have the following format:
 
+**Transcript Object**
```

Comment:
Not sure these titles are even necessary. I think we could just have the json follow